coursier.bzl

```diff
@@ -263,7 +278,7 @@ def generate_imports(repository_ctx, dep_tree, srcs_dep_tree = None):
             reverse_dep_coords = [reverse_dep["coord"] for reverse_dep in reverse_deps]
             reverse_dep_pom_paths = [
                 repository_ctx.path(reverse_dep["file"].replace(".jar", ".pom").replace(".aar", ".pom"))
-                for reverse_dep in reverse_deps
+                for reverse_dep in reverse_deps if reverse_dep["file"]
```

Comment:
I think the "if reverse_dep["file"] part isn't necessary because the previous elif clause will catch the pom only dependencies which are the ones that don't have a file

Reply:
Great catch. Done.